torch/_inductor/async_compile.py

```diff
@@ -219,7 +220,23 @@ def remove_future(kernel_src: str) -> None:
             del CompiledTritonKernels._cache[key]
 
 
+@contextlib.contextmanager
+def async_compile_pool_manager():
+    """
+    Context manager to quiesce the subproc pool at the end of compilation, i.e.,
+    when dynamo is done.
+    """
+    try:
+        yield
+    finally:
+        AsyncCompile.quiesce()
```

Comment:
In a later optimization it might be worth starting a timer which quiesces the pool if there has been no activity for some amount of time (5s?) just to keep it from flapping...

Reply:
A timer seems fine as well.